### PR TITLE
fix the edit/refresh chart

### PR DESCRIPTION
**Pull request by devoncarew**
- fix the edit/refresh chart (issues constructing the data for the charts, and a NPE with tooltips)

### CODE REVIEW

**Comment by devoncarew** [comment]
@sethladd @yjbanov

File: web/charts.dart [hunk 1]
```diff
@@ -264,3 +268,9 @@ DivElement div(String text, { String className }) {
     element.className = className;
   return element;
 }
+
+String _comma(String str) {
```
Comment (yjbanov) [inline web/charts.dart]: Had to look at the call site to figure out what this function does. Perhaps take `int` as argument and name it something like `_formatWithThousandsSeparator`?

**Comment by yjbanov** [comment]
LGTM

File: web/charts.dart (hunk 1 above)
Comment (devoncarew) [inline web/charts.dart]: sg

**Pull request merged**